src/foam/nanos/crunch/README.md

```diff
@@ -0,0 +1,83 @@
+# CRUNCH User Guide and Documentation
+
+&nbsp;
+
+### CRUNCH - Continuous Reactive User Nano-Capability Hierarchy
+
+CRUNCH is a [Knowledge-Based Configuration System](https://en.wikipedia.org/wiki/Knowledge-based_configuration) that configures a user's account with various capabilities based upon the features and functionalities.
+Using CRUNCH we model all the potential capabilities of a user along with various compliance, onboarding, security, and approval requirements of those capabilities. Those models form a knowledge-graph which allows CRUNCH to query for set of capabilities and receive sets of onboarding requirements.
+CRUNCH uses this knowledge graph to dynamically generate onboarding wizards for the particular configuration or set of capabilities that a user has requested, as well as dynamically add capabilities to a user's account at run-time and provide a gradual onboarding experience.
+For more information on CRUNCH as a solution to the onboarding problem, and some concrete use cases of CRUNCH, please visit the [CRUNCH wiki](https://github.com/nanoPayinc/NANOPAY/wiki/CRUNCH)
```

Comment:
This link is only available inside nanopay. If none of the material is proprietary, then maybe we should move to the FOAM repo.

Reply:
I'm not sure that it's non-proprietary, will remove this for now